pythermalcomfort/utilities.py

```diff
@@ -1245,3 +1245,84 @@ class DefaultSkinTemperature(NamedTuple):
     right_thigh: float = 34.3
     right_leg: float = 32.8
     right_foot: float = 33.3
+
+
+def scale_windspeed(
+    va: float | list[float], h: float | list[float], z0: float = 0.01
```

Comment:
Thank you for implement this, the value it should not just be a float but the user should also be able to pass an array, as long as they have the same lenght as va and h.